wavedrom/package.json

```diff
@@ -17,7 +17,8 @@
   "dependencies": {
     "json5": "2.2.0",
     "onml": "2.1.0",
-    "wavedrom": "2.6.8"
+    "wavedrom": "2.6.8",
+    "bit-field": "1.3.6"
```

Comment:
`bit-field` is already a dependency of `wavedrom`, see: https://github.com/wavedrom/wavedrom/blob/7b78999021378bf49ebc1dafa31baf3d09f20a64/package.json#L61

Reply:
So, I guess there is nothing to do then? I'm confused now. Maybe it didn't work because the yuzutech/kroki container I pulled has an older version of wavedrom?

Reply:
Probably, the current Docker image (0.11) is using WaveDrom 2.3.2.

Do you have a sample diagram so we can add a test to make sure that it's working on the master branch with WaveDrom 2.6.8 ?

Reply:
Yes, here is one, taken from https://observablehq.com/@drom/bitfield-testbox?collection=@drom/bitfield:

```
{reg: [
    {name: "IPO",   bits: 8, attr: "RO"},
    {               bits: 7},
    {name: "BRK",   bits: 5, attr: ["RW"], type: 4},
    {name: "CPK",   bits: 1},
    {name: "Clear", bits: 3, type: 5},
    {               bits: 8}
]}
```

Reply:
This example is already working today with Kroki 0.11:

![image](https://user-images.githubusercontent.com/333276/113259322-4811a900-92cd-11eb-95b8-fe4340c63eb1.png)

Reply:
This simple one with rotate should do it:
```
{reg: [
  {name: "long_label", bits: 1, rotate: -90},
  {name: '"longer_label", bits: 3, rotate: -45},
  {bits: 28}
],
config: {vspace: 150}
}
```

Reply:
Thanks, I can confirm that this definition does not work on Kroki 0.11, I will try the same definition against master branch (WaveDrom 2.6.8)